### DISCUSSION TITLE
Convert this repo to an organizaiton

### DISCUSSION THREAD

**Discussion by bytrangle** [super-productivity/super-productivity#2885] [Ideas]
Hello. Thanks for all the hard work you've put into this project, making it a truly practical, usable to-do app. Here's why I think it may be good to transfer it a an organization:
- It signifies that this is a serious project. It's not that a repo under an individual account can't be serious and impactful, but from the perspective of aspiring contributors, a repo under an organization signifies higher commitment. I'm more likely to prioritize contributing to repos by organizations. I know it's not always true, having more contributors is still better than having few.
- You are still the admin of the project and in charge of its direction.
- You have more flexibility in managing collaborators' access. Let more people help you.
What do you think?

**Comment by johannesjo** [comment]
Thanks for opening this up the discussion! To be honest super productivity is also kind of a reference for me as a freelancer. Having my username attached to it helps to quickly make the connection when people visit my github profile. Transferring it an organization would require much more effort to make the connection and most people won't care enough to do so. You're making valid points, but imho the benefits of it don't outweigh the disadvantages (takes work, might migrations tend to cause unintended problems and the project won't be a good reference any more for me personally). At least at this point of time :) I hope you can understand my reasoning.
